ProjectDataCore/Pages/Admin/NavBar/ModifyNavBar.razor.cs

```diff
@@ -0,0 +1,87 @@
+﻿using ProjectDataCore.Data.Services.Nav;
+using ProjectDataCore.Data.Structures.Nav;
+
+namespace ProjectDataCore.Pages.Admin.NavBar
+{
+    public partial class ModifyNavBar : ComponentBase, IDisposable
+    {
+        [Inject] private INavModuleService _navModuleService { get; set; }
```

Comment:
For consistency, make this a public property, please (and the name formatting that goes with it).